regress/unittests/test_helper/fuzz.c

```diff
@@ -160,7 +160,7 @@ dump(u_char *p, size_t len)
 	size_t i, j;
 
 	for (i = 0; i < len; i += 16) {
-		fprintf(stderr, "%.4zd: ", i);
```

Comment:
Revert changes in readpass.c as there are no changes.

Reply:
Reverted readpass.c. Was there another comment for this file - fuzz.c?